orientdb/pom.xml

```diff
@@ -63,21 +63,25 @@
 
         <dependency>
             <groupId>com.orientechnologies</groupId>
-            <artifactId>orientdb-core</artifactId>
+            <artifactId>orientdb-core</artifactId>            
```

Comment:
This kind of chges should not be included in a PR. I don't think you need to change this file BTW.
Is there something you need to add to this POM to work with OrientDB?